.gitignore

```diff
@@ -1,3 +1,4 @@
 /vendor/
 build/
 composer.lock
+.phpunit.result.cache
```

Comment:
Could we not add this please? I think this file, and other artifacts or IDE files (`.idea` for example) should ideally be in the `~/.gitignore` file, so it does not need to be added to each project.

Reply:
OK. Reverted.